motion_config.html

```diff
@@ -947,6 +947,12 @@ <h2><a name="Configuration_OptionsAlpha"></a>  Configuration Options-Listed Alph
           <td align="left">sql_query_start</td>
           <td align="left"><a href="#sql_query_start" >sql_query_start</a></td>
         </tr>
+        <tr>
+          <td height="17" align="left"></td>
+          <td align="left"></td>
+          <td align="left">sql_query_stop</td>
```

Comment:
Can you review whether it is correct to include this in the third column?  From memory, I thought the third column was for version 4.1 and if it is a new option, then we would only list it in the last column (current version)

Reply:
You are right, it's wrong, will fix.